haystack/components/embedders/sentence_transformers_text_embedder.py

```diff
@@ -14,7 +14,8 @@
 @component
 class SentenceTransformersTextEmbedder:
     """
-    A component for embedding strings using Sentence Transformers models.
```

Comment:
All other components in Haystack use one sentence for this docstring and start with `A component`. If we change only the class `SentenceTransformersTextEmbedder` it's not really standardized.

Reply:
the goal is also to shorten the docstrings and not repeat what's obvious, that's why I did away with "A component for.."

Reply:
@julian-risch "A component" is used often but is not a standard (see [DocumentWriter](https://docs.haystack.deepset.ai/reference/document-writers-api), [CacheChecker](https://docs.haystack.deepset.ai/reference/caching-api), [converters](https://docs.haystack.deepset.ai/reference/converters-api#pypdftodocument) for examples), I would say that the shorter style that describes what component is doing looks quite good.

Reply:
@julian-risch are you OK with the new format? can I resolve this?